lib/src/color_picker.dart

```diff
@@ -272,26 +272,31 @@ class _ColorPickerState extends State<ColorPicker> {
     );
   }
 
-  Widget _buildDropdown2() {
-    return SizedBox(
-      height: 38,
-      child: DecoratedBox(
-        decoration: BoxDecoration(
-          border: Border.all(color: Theme.of(context).dividerColor),
-          borderRadius: const BorderRadius.all(
-            Radius.circular(3.0),
+  Widget _buildDropdownPortraitMode() {
+    return Padding(
+      padding: const EdgeInsets.symmetric(vertical: 6.0),
+      child: SizedBox(
+        height: 38,
+        child: DecoratedBox(
+          decoration: BoxDecoration(
+            border: Border.all(color: Theme.of(context).dividerColor),
+            borderRadius: const BorderRadius.all(
```

Comment:
Maybe the `circular` constuctor will be better.

Reply:
Done (makes it not const any more)